doc/userguide/capture-hardware/ebpf-xdp.rst

```diff
@@ -64,12 +71,17 @@ Kernel
 
 You need to run a kernel 4.13 or newer.
 
-Clang
-~~~~~
+Clang and dependencies
+~~~~~~~~~~~~~~~~~~~~~~
 
 Make sure you have clang (>=3.9) installed on the system  ::
 
- sudo apt-get install clang
+ sudo apt install clang
+
+Some i386 headers will also be needed as eBPF is not x86_64 and some included headers
+are architecture specific ::
+
+ sudo apt install  libc6-dev-i386 --no-install-recommends
```

Comment:
double spaces between install and libc6.

Might also be helpful to link so some explanation why it's not x86_64 and if there is a way for 64-Bit only systems that don't have 32-bit libs anymore.